### PR TITLE
Add View docs

### PR DESCRIPTION
**Pull request by jknaudt21**
### Description of Change

Add docs for View

### Issues Fixed

Related to #3960

### CODE REVIEW

File: src/Controls/src/Core/View/View.cs [hunk 1]
```diff
@@ -186,33 +199,55 @@ protected internal override void ChangeVisualState()
 			PointerGestureRecognizer
 				.SetupForPointerOverVSM(this, (result) => SetPointerOver(result), ref _recognizerForPointerOverState);
 
-		/// <include file="../../docs/Microsoft.Maui.Controls/View.xml" path="//Member[@MemberName='GetChildElements']/Docs/*" />
+		/// <summary>
+		/// Gets the child elements that are visually beneath the specified <paramref name="point" />.
+		/// </summary>
+		/// <param name="point">The point under which to search for child elements.</param>
+		/// <returns> All child elements visually beneath <paramref name="point"/>.</returns>
```
Comment (jfversluis) [inline src/Controls/src/Core/View/View.cs:206]:
```suggestion
		/// <returns>All child elements visually beneath <paramref name="point"/>.</returns>
```

File: src/Controls/src/Core/View/View.cs [hunk 2]
```diff
@@ -186,33 +199,55 @@ protected internal override void ChangeVisualState()
...
 		public virtual IList<GestureElement> GetChildElements(Point point)
 		{
 			return null;
 		}
 
-		/// <include file="../../docs/Microsoft.Maui.Controls/View.xml" path="//Member[@MemberName='HorizontalOptions']/Docs/*" />
+		/// <summary>
+		/// Gets or sets the <see cref="LayoutOptions" /> that define how the element gets laid in a layout cycle. This is a bindable property.
```
Comment (jfversluis) [inline src/Controls/src/Core/View/View.cs:213]:
```suggestion
		/// Gets or sets the <see cref="LayoutOptions" /> that define how the element gets laid out in a layout cycle. This is a bindable property.
```

File: src/Controls/src/Core/View/View.cs [hunk 3]
```diff
@@ -186,33 +199,55 @@ protected internal override void ChangeVisualState()
...
+		/// </summary>
 		public Thickness Margin
 		{
 			get { return (Thickness)GetValue(MarginProperty); }
 			set { SetValue(MarginProperty, value); }
 		}
 
-		/// <include file="../../docs/Microsoft.Maui.Controls/View.xml" path="//Member[@MemberName='VerticalOptions']/Docs/*" />
+		/// <summary>
+		/// Gets or sets the <see cref="LayoutOptions" /> that define how the element gets laid in a layout cycle. This is a bindable property.
```
Comment (jfversluis) [inline src/Controls/src/Core/View/View.cs:235]:
```suggestion
		/// Gets or sets the <see cref="LayoutOptions" /> that define how the element gets laid out in a layout cycle. This is a bindable property.
```

**Review by jfversluis — CHANGES_REQUESTED** [review]

File: src/Controls/src/Core/View/View.cs [hunk 4]
```diff
@@ -150,14 +158,19 @@ void RemoveItems(IEnumerable<IElementDefinition> elements)
 			};
 		}
 
-		/// <include file="../../docs/Microsoft.Maui.Controls/View.xml" path="//Member[@MemberName='GestureRecognizers']/Docs/*" />
+		/// <summary>The collection of gesture recognizers associated with this view.</summary>
+		/// <remarks>
+		/// Adding items to this collection will associate gesture events with this element.
+		/// This is not necessary for elements that natively support certain gestures. For example, <see cref="Button"/> already supports taps.
```
[the remark this answers is not in the document]
Comment (jknaudt21) [reply src/Controls/src/Core/View/View.cs]: Added this commit to address this: https://github.com/dotnet/maui/pull/17220/commits/068b75ac8fb410cf826ada62e6117fdbc7b407e9

**Review by jfversluis — APPROVED** [review]

**Comment by jfversluis** [comment]
Suggesting backport because it's relatively safe to do and gives up more coverage on the docs in vCurrent

**Pull request merged**